sklearn/neighbors/_binary_tree.pxi

```diff
@@ -1158,16 +1159,16 @@ cdef class BinaryTree:
         self._update_sample_weight(n_samples, sample_weight_arr)
         self._update_memviews()
 
-    def get_tree_stats(self):
+    def _get_tree_stats(self):
```

Comment:
Understood. It should either use a deprecation warning or add docstring to the methods. Thanks for the review. I will modify this.

Reply:
Since I cannot find these 4 methods called by internal functions, I presume they are convenient methods for users. Therefore, I add docstring to them.